apps/prairielearn/src/pages/instructorQuestions/instructorQuestions.ts

```diff
@@ -3,18 +3,59 @@ import asyncHandler from 'express-async-handler';
 import fs from 'fs-extra';
 
 import * as error from '@prairielearn/error';
+import { flash } from '@prairielearn/flash';
 
 import { InsufficientCoursePermissionsCardPage } from '../../components/InsufficientCoursePermissionsCard.js';
+import { config } from '../../lib/config.js';
 import { getCourseFilesClient } from '../../lib/course-files-api.js';
 import { getCourseOwners } from '../../lib/course.js';
 import { features } from '../../lib/features/index.js';
+import { EXAMPLE_COURSE_PATH } from '../../lib/paths.js';
 import { selectCourseInstancesWithStaffAccess } from '../../models/course-instances.js';
 import { selectQuestionsForCourse } from '../../models/questions.js';
+import { loadQuestions } from '../../sync/course-db.js';
 
 import { QuestionsPage } from './instructorQuestions.html.js';
 
 const router = Router();
 
+let cachedTemplateQuestionOptions: { qid: string; title: string }[] | null = null;
+
+/**
+ * Get a list of template question qids and titles that can be used as starting points for new questions.
+ */
+async function getTemplateCourseQuestionOptions(): Promise<{ qid: string; title: string }[]> {
+  if (!config.devMode) {
+    // Check if the template questions are cached
+    if (cachedTemplateQuestionOptions) {
+      return cachedTemplateQuestionOptions;
+    }
+  }
+
+  const questions = await loadQuestions(EXAMPLE_COURSE_PATH);
+
+  const templateQuestions: { qid: string; title: string }[] = [];
+
+  for (const qid of Object.keys(questions)) {
+    const question = questions[qid];
+    const qidParts = qid.split('/');
+    if (qidParts.length > 0 && qidParts[0] === 'template' && question?.data?.title) {
+      templateQuestions.push({ qid, title: question.data.title });
+    }
+  }
```

Comment:
Nit, this could be simplified a little bit:

```suggestion
  for (const [qid, question] of Object.entries(questions)) {
    if (qid.startsWith('template/') && question?.data?.title) {
      templateQuestions.push({ qid, title: question.data.title });
    }
  }
```